packages/inspire/index.js

```diff
@@ -26,12 +26,17 @@ const Valaa = getGlobal().Valaa || (getGlobal().Valaa = {});
 
 Valaa.getURIQueryField = getURIQueryField;
 
+
 Valaa.createInspireGateway = function createInspireGateway (...revelations: any[]) {
   const gatewayPromise = Valaa.createGateway(...revelations);
   return new Promise(resolve =>
       document.addEventListener("DOMContentLoaded", () => { resolve(gatewayPromise); }));
 };
 
+Valaa.createGateway = function createPerspire (...revelations: any[]) {
```

Comment:
How does this work without creating an infinite loop? :D 
Edit: nm I'm silly: it's just immediately overwritten by line 40, so these lines are dead code.
This probably should be named createPerspireGateway (both functions).

The work that createInspireGateway does on top of createGateway is that it adds the hook for when DOM is loaded, to signal that the second stage of initialization can start. 
I don't know yet how this should be handed on the server side tbh.